src/sign/network.tf

```
@@ -0,0 +1,15 @@
+data "azurerm_virtual_network" "vnet_common" {
+  name                = "io-p-vnet-common"
+  resource_group_name = "io-p-rg-common"
+}
+
+module "io_sign_snet" {
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.13.1"
+  name                 = "${local.project}-snet"
+  resource_group_name  = azurerm_resource_group.backend_rg.name
+  virtual_network_name = data.azurerm_virtual_network.vnet_common.name
+  address_prefixes     = ["10.0.101.0/24"]
+
+  enforce_private_link_endpoint_network_policies = true
+  service_endpoints                              = ["Microsoft.Web"]
```

```suggestion
  service_endpoints                              = ["Microsoft.Web", "Microsoft.AzureCosmosDB",]
```
you need to access to cosmos db so u need also Microsoft.AzureCosmosDB service endpoint